cpp/include/cudf/wrappers/bool.hpp

```diff
@@ -151,7 +151,7 @@ struct bool8 {
   }
 
 private:
-  value_type value{0};
+  value_type value;
```

Comment:
What was the problem here?

Reply:
The problem is if you try to do this:
```
struct foo{
    //int v{}; // this doesn't work
    int v; // this works
};


__global__ void kernel(float f1, float f2, bool * out){
   __shared__ foo foos[32];
   foos[0] = foo{13};
}
```
CUDA doesn't let you have a default initializer for shared memory. 

https://godbolt.org/z/k7-Hmq

Reply:
Yikes, that's right!